readthedocs/settings/base.py

```diff
@@ -25,7 +27,27 @@
 
 
 _ = gettext = lambda s: s
-log = logging.getLogger(__name__)
+log = structlog.get_logger(__name__)
+
+def send_to_newrelic(logger, log_method, event_dict):
```

Comment:
I'd like to see this outside of `settings/base`, perhaps in a `utils` or something.